src/components/playlist-countdown/playlist-countdown.js

```diff
@@ -135,7 +135,7 @@ class PlaylistCountdown extends Component {
    * @returns {boolean} shouldComponentUpdate
    */
   shouldComponentUpdate(nextProps: Object): boolean {
-    return !(nextProps.isSeeking || this.props.isPlaybackEnded);
+    return !!(this.props.duration && !nextProps.isSeeking && !this.props.isPlaybackEnded);
```

Comment:
i never understood the purpose of doing a "!!" isn't it just clear without it?

Reply:
@RoyBregman This is to make `0` to `false` (`shouldComponentUpdate` expects to `boolean`)

Reply:
Agree that it's more clear to do this as `this.props.duration > 0`

Reply:
done